Put metric before label value in error message when 'label value too long' (#3982)
cortexproject/cortex: CHANGELOG.md

  * `-alertmanager.cluster.advertise-address` instead of `-cluster.advertise-address`
  * `-alertmanager.cluster.peers` instead of `-cluster.peer`
  * `-alertmanager.cluster.peer-timeout` instead of `-cluster.peer-timeout`

> * [CHANGE] Put metric before label value in error message when `labelValueTooLong`. #3982

Reviewer: ```suggestion
* [ENHANCEMENT] Put metric before label value in the "label value too long" error message. #3982
```
